src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Epas/BatchEpaRequestValidator.cs

```diff
@@ -57,7 +57,7 @@ public BatchEpaRequestValidator(IStringLocalizer<BatchEpaRequestValidator> local
                     {
                         context.AddFailure(new ValidationFailure("CourseOption", "No course option available for this Standard and version. Must be empty"));
                     }
-                    else if (standardOptions != null && standardOptions.HasOptions() && standardOptions.CourseOption.All(a => a.IndexOf(m.CourseOption, StringComparison.OrdinalIgnoreCase) == -1))
+                    else if (standardOptions.HasOptions() && standardOptions.CourseOption.All(a => a.IndexOf(m.CourseOption, StringComparison.OrdinalIgnoreCase) == -1))
```

Comment:
the if above checks if it's null, so by implicit logic, it's not null here so can be removed